shared/js/bluetooth_helper.js

```diff
@@ -175,11 +175,6 @@ var BluetoothHelper = function() {
     },
 
     getConnectedDevicesByProfile: function(profileID, cb) {
-      if (_v2) {
-        console.log('getConnectedDevicesByProfile API is deprecated');
```

Comment:
Since adapter.getConnectedDevices api is still exist, we could just remove the false alert

https://dxr.mozilla.org/mozilla-central/source/dom/webidl/BluetoothAdapter.webidl?from=BluetoothAdapter.webidl&offset=0#108